pom.xml

```diff
@@ -64,10 +64,10 @@
   </scm>
 
   <properties>
-    <revision>2.12.5</revision>
+    <revision>2.12.4-1</revision>
```

Comment:
so we can release a version and keep the versions aligned to the jackson version

Reply:
can we do:
```suggestion
    <revision>${jackson.version}-1</revision>
```

Reply:
I honestly don't know.  I would love to do that if it works.
Do you know if that has been tested anywhere else?

Reply:
caffeine-api uses a slightly different approach ->
https://github.com/jenkinsci/caffeine-api-plugin/blob/master/pom.xml#L22-L23

https://github.com/jenkinsci/caffeine-api-plugin/blob/master/pom.xml#L44-L47

but it is using the CD for plugins approach so that number is just ignored.

Reply:
jjwt-api apparently https://github.com/jenkinsci/jjwt-api-plugin/blob/master/pom.xml#L37

Reply:
that's also using CD for plugins so I expect m-r-p to trash it after a release (this plugin is not yet using CD flow)

lets see how it goes - if it does not work it should just revert to a string without interpolation...

Reply:
FYI: it did not work -> https://github.com/jenkinsci/jackson2-api-plugin/blob/068492eb889a909cf0d51da661786999ce0c425c/pom.xml#L67

I tried to be clever as well...

`What is the new development version for "Jackson 2 API Plugin"? (org.jenkins-ci.plugins:jackson2-api) 2.12.4-2-SNAPSHOT: : ${jackson.version}-2-SNAPSHOT`

Still nothing ventured nothing gained.